tasks/summarize-zap-results/task.rb

```diff
@@ -11,8 +11,12 @@ def project_name
 end
 
 puts "Comparing last results to current results..."
-ZAPResultSetComparator.write_summary(last_run_dir, current_run_dir, output_dir, project_name)
+comparator = ZAPResultSetComparator.new(project_name, last_run_dir, current_run_dir)
+if comparator.no_change?
```

Comment:
Nope, works for me! Maybe just link to https://github.com/cloudfoundry-community/slack-notification-resource#parameters and mention it in an inline comment?